theme/style.css

```diff
@@ -24,3 +24,20 @@ img {
 img {
   width: 90%;
 }
+
+/* Background pattern from Toptal Subtle Patterns */
+#kaidiv {
+ font-family: Arial, Helvetica, sans-serif;
+  background-image: url(../assets/skulls.png);
+}
+
+h3 .kaipcenter,
```

Comment:
This isn't doing what you think it's doing. Remember the difference between the indirect descendant selector (e.g. `h3 .kaipcenter`) and the class selector (e.g. `h3.kaipcenter`)